### PR TITLE
Fix lotus bench error

### LINKED ISSUE #4304: lotus-bench error!!!
https://github.com/filecoin-project/lotus/blob/00620aac572af059bd0daab6d0dcfadbaa7beae1/cmd/lotus-bench/main.go#L617
https://github.com/filecoin-project/lotus/blob/00620aac572af059bd0daab6d0dcfadbaa7beae1/cmd/lotus-bench/main.go#L628
Should be:
abi.SectorID{Miner: mid, Number: **i**} 
1 should be i

### PR DESCRIPTION
**Pull request by zgfzgf**
fix #4304

### CODE REVIEW

**Comment by zgfzgf** [comment]
Hi @magik6k 
Why does test fail to modify cmd/lotus-bench/main.go?
Coupling?

ci/circleci: test-short — Your tests failed on CircleCI

**Comment by zgfzgf** [comment]
local:

go test -v -run TestAPIDealFlow node/node_test.go
2020-10-11T17:31:51.017+0800	INFO	pubsub	go-libp2p-pubsub@v0.3.6/pubsub.go:619	pubsub processloop shutting down
PASS
ok  	command-line-arguments	410.284s

File: cmd/lotus-bench/main.go [hunk 1]
```diff
@@ -428,21 +462,21 @@ var sealBenchCmd = &cli.Command{
...
-			fmt.Printf("----\nresults (v27) (%d)\n", sectorSize)
+			fmt.Printf("----\nresults (v28) SectorSize:(%d), SectorNumber:(%d)\n", sectorSize, sectorNumber)
 			if robench == "" {
-				fmt.Printf("seal: addPiece: %s (%s)\n", bo.SealingResults[0].AddPiece, bps(bo.SectorSize, bo.SealingResults[0].AddPiece)) // TODO: average across multiple sealings
-				fmt.Printf("seal: preCommit phase 1: %s (%s)\n", bo.SealingResults[0].PreCommit1, bps(bo.SectorSize, bo.SealingResults[0].PreCommit1))
-				fmt.Printf("seal: preCommit phase 2: %s (%s)\n", bo.SealingResults[0].PreCommit2, bps(bo.SectorSize, bo.SealingResults[0].PreCommit2))
-				fmt.Printf("seal: commit phase 1: %s (%s)\n", bo.SealingResults[0].Commit1, bps(bo.SectorSize, bo.SealingResults[0].Commit1))
-				fmt.Printf("seal: commit phase 2: %s (%s)\n", bo.SealingResults[0].Commit2, bps(bo.SectorSize, bo.SealingResults[0].Commit2))
-				fmt.Printf("seal: verify: %s\n", bo.SealingResults[0].Verify)
+				fmt.Printf("seal: addPiece: %s (%s)\n", bo.SealingSum.AddPiece, bps(bo.SectorSize, bo.SectorNumber, bo.SealingSum.AddPiece))
```
Comment (magik6k) [inline cmd/lotus-bench/main.go:467]: We probably want to print the average / slowest time here

**Review by magik6k — COMMENTED** [review]
Thanks for the PR! 

Changes generally look good, just 1 comment

File: cmd/lotus-bench/main.go (hunk 1 above)
Comment (zgfzgf) [reply cmd/lotus-bench/main.go:467]: old code it's the first average

**Comment by zgfzgf** [comment]
hi @magik6k 
This is a separate module and will not affect other programs
Early merger

**Review by magik6k — COMMENTED** [review]
Merge conflicts :/

**Comment by zgfzgf** [comment]
@magik6k test ok 
please merge

**Review by magik6k — APPROVED** [review]
Looks good, thanks

**Pull request merged**